system/jhtree/ctfile.hpp

```diff
@@ -140,6 +143,17 @@ struct jhtree_decl NodeHdr
 //#pragma pack(4)
 #pragma pack(pop)
 
+// Additional header info after the standard node header, for POC Split Nodes
+
+#pragma pack(push,1)
```

Comment:
question: is there any point/effect of the pack, with this struct having 1 member?

Reply:
In case I add more? Since this is a template for adding other compression methods, that's not unlikely...